nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/java/org/nuxeo/ftest/server/hotreload/ITDevHotReloadTest.java

```diff
@@ -153,4 +158,30 @@ public void testHotReloadPageProvider() {
         assertEquals(1, nbDocs);
     }
 
+    @Test
+    public void testHotReloadPermission() {
+        RestHelper.createUser("john", "doe");
+        String docId = RestHelper.createDocument("/", "File", "file");
+        // we need these permissions in order to correctly check the hot reload
```

Comment:
Why?

Reply:
because the hotReloadPermission includes them (?) it's remove though, not removeChildren, so not sure...

Reply:
I can add details to my comment. Here is the point: when you want to remove/delete a document from Nuxeo, you need to have the `Remove` permission on this document but you also need `RemoveChildren` on the parent. And as we need to resolve the parent to check the second permission, we need to have the `Read` permission on it.